.github/workflows/rebuild-trivy-bundle.yaml

```diff
@@ -0,0 +1,261 @@
+        if: needs.determine-if-workflow-should-run.outputs.WORKFLOW_SHOULD_RUN
+        run: |
+          sudo mkdir -p //usr/local/go/src/github.com/mesosphere/dkp-insights
+          sudo chmod  -R  0777 //usr/local/go/src/github.com/mesosphere/dkp-insights
+
+      - name: Clone dkp-insights Repository
+        if: needs.determine-if-workflow-should-run.outputs.WORKFLOW_SHOULD_RUN
+        uses: actions/checkout@v3
+        with:
+          repository: mesosphere/dkp-insights
+          ref: main
+          submodules: 'true'
+          path: dkp-insights
+          token: ${{ secrets.MERGEBOT_TOKEN }}
+
+      - name: Move dkp-insights Repository to GOPATH
+        if: needs.determine-if-workflow-should-run.outputs.WORKFLOW_SHOULD_RUN
+        run: |
+          mv ./dkp-insights/* ./dkp-insights/.[!.]* ./dkp-insights/..?* //usr/local/go/src/github.com/mesosphere/dkp-insights/ || :
+          cd //usr/local/go/src/github.com/mesosphere/dkp-insights
+          ls -a
+          
+      - name: Fetch All Branches from dkp-insights Repository
+        if: needs.determine-if-workflow-should-run.outputs.WORKFLOW_SHOULD_RUN
+        working-directory: //usr/local/go/src/github.com/mesosphere/dkp-insights
+        run: git fetch
+
+      - name: Set Git Global Username and Email
+        if: needs.determine-if-workflow-should-run.outputs.WORKFLOW_SHOULD_RUN
+        run: |
+          git config --global user.email "ci-mergebot@d2iq.com"
+          git config --global user.name "d2iq-mergebot"
+
+      - name: Download Artifact with dkp-insights Release Branches and their Updated trivy-bundles Image Tags
+        if: needs.determine-if-workflow-should-run.outputs.WORKFLOW_SHOULD_RUN
+        uses: actions/download-artifact@v3
+        with:
+          name: updated-trivy-bundles-image-tags
+      
+      - name: Move updated-trivy-bundles-image-tags artifact to dkp-insights repository
+        if: needs.determine-if-workflow-should-run.outputs.WORKFLOW_SHOULD_RUN
+        run: mv updated-trivy-bundles-image-tags.txt //usr/local/go/src/github.com/mesosphere/dkp-insights
+      
+      - name: Install Hub (wrapper for git for opening PRs)
+        if: needs.determine-if-workflow-should-run.outputs.WORKFLOW_SHOULD_RUN
+        run: sudo apt install hub
+      
+      - name: Update trivy-bundles Image Tags in Each dkp-insights Release Branch and Open a Pull Request
+        if: needs.determine-if-workflow-should-run.outputs.WORKFLOW_SHOULD_RUN
+        working-directory: //usr/local/go/src/github.com/mesosphere/dkp-insights
+        env:
+          GITHUB_TOKEN: ${{ secrets.MERGEBOT_TOKEN }}
+        run: |
+          echo "setup environment"
+          export PATH=$PATH:/usr/local/go/bin
+          echo "GOPATH=/usr/local/go" >> ~/.bashrc
+          export GOPATH=/usr/local/go
+          export IR=${GOPATH}/src/github.com/mesosphere/dkp-insights
+          export PATH=${IR}/.local/tools:${PATH}
```

Comment:
(same as previous comment, we don't build `dkp-insights` anymore)

Reply:
You're right.  I'll correct them all now